components/keystone/aio-values.yaml

```diff
@@ -19,6 +19,11 @@ bootstrap:
     # give 'argoworkflow' 'admin' over the 'baremetal' project
     openstack role add --user-domain infra --project-domain infra --user argoworkflow --project baremetal admin
 
+    # create 'flavorsync' user to allow synchronization of the flavors to nova
+    openstack user create --or-show --domain service --password abcd1234 flavorsync
+    openstack role create --or-show --domain service flavorsync
+    openstack role add --user-domain service --user flavorsync flavorsync
```

Comment:
We really shouldn't be creating random users with random passwords. We should be creating application credentials for our service account.

Reply:
Agreed. This is just an example that is supposed to be overriden in the deploy repository (just as it is for the `argoworkflow` and `monitoring` users above and below.